lightstep/sdk/metric/exporters/otlp/otelcol/client.go

```diff
@@ -175,7 +171,7 @@ func NewExporter(ctx context.Context, cfg Config) (metric.PushExporter, error) {
 	}
 
 	bset := processor.Settings{
-		ID:                component.NewID(component.MustNewType("otel_sdk_metric_batch")),
```

Comment:
just updating these component names to make the tests pass, I don't think they actually mean anything. (And if they do mean something, they were wrong anyways, so)